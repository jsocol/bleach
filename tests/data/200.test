<div id="134"><xmp>
<%
</xmp>
<img alt='%></xmp><img src=xx onerror=alert(134)//'>

<script>
x='<%'
</script> %>/
alert(2)
</script>

XXX
<style>
*['<!--']{}
</style>
-->{}
*{color:red}</style>//["'`-->]]>]</div>
