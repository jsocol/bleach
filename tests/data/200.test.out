<div id="134">
&lt;%

<img alt="%></xmp><img src=xx onerror=alert(134)//">

 %&gt;/
alert(2)


XXX
<style>
*['<!--']{}
</style>
--&gt;{}
*{color:red}//["'`--&gt;]]&gt;]</div>
